bodhi/tests/server/views/test_generic.py

```diff
@@ -53,6 +53,16 @@ def test_home(self):
         self.assertIn('Log out', res)
         self.assertIn('Fedora Update System', res)
 
+    @unittest.skipIf(six.PY3, 'Not working with Python 3 yet')
```

Comment:
This test should work with Python 3 I think - let's see if we can remove this decorator and get the test to pass.